### PR TITLE
Same site cookie config

### PR DESCRIPTION
**Pull request by gregw**
[Same  site](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-03#section-4.1.2.7) cookie config for https://github.com/eclipse-ee4j/servlet-api/pull/271

### CODE REVIEW

File: jakartaee9/src/web-common_4_0.xsds [hunk 1]
```diff
@@ -972,6 +972,19 @@
...
       </xsd:element>
+      <xsd:element name="same-site"
+                   type="jakartaee:cookie-sameSiteType"
+                   minOccurs="0">
+        <xsd:annotation>
+          <xsd:documentation>
+
+            Specifies whether any session tracking cookies created
+            by this web application will be marked with a Same-Site
+	    attribute
```
Comment (bshannon) [inline jakartaee9/src/web-common_4_0.xsds]: There's some indentation or space/tab confusion here.

File: jakartaee9/src/web-common_4_0.xsds [hunk 2]
```diff
@@ -1084,6 +1097,30 @@
...
   </xsd:complexType>
 
+<!-- **************************************************** -->
+
+  <xsd:complexType name="cookie-sameSiteType">
+    <xsd:annotation>
+      <xsd:documentation>
+
+        This type defines the values for the Same-Site cookie 
+	attribute as defined by
```
Comment (bshannon) [inline jakartaee9/src/web-common_4_0.xsds]: More indentation confusion.

File: jakartaee9/src/web-common_4_0.xsds (hunk 1 above)
Comment (gregw) [reply jakartaee9/src/web-common_4_0.xsds]: My editor put a tab in there!  fixed

File: jakartaee9/src/web-common_4_0.xsds (hunk 2 above)
Comment (gregw) [reply jakartaee9/src/web-common_4_0.xsds]: fixed

**Comment by gregw** [comment]
There was no cookie-config test at all, so I added simple one

**Comment by gregw** [comment]
So judging by comments on https://github.com/eclipse-ee4j/servlet-api/pull/271, this will not be for EE9, but would be good to have in a EE9.1 (is there such a thing?) to support a servlet-1pi 5.1 soon after.

**Comment by arjantijms** [comment]
> would be good to have in a EE9.1 (is there such a thing?) to support a servlet-1pi 5.1 soon after

There's no EE 9.1 planned, the next will be EE 10. EE 10 could potentially be pretty fast after EE 9.

**Comment by ivargrimstad** [comment]
Fixed by https://github.com/eclipse-ee4j/jakartaee-schemas/pull/25

**Pull request closed without merging**